src/views/contact/ContactPage.js

```diff
@@ -56,25 +66,19 @@ const ContactPage = () => {
             <tr>
               <td>Inhoudelijk contactpersoon</td>
               <td>
-                <a
-                  className={styles.EmailLink}
-                  href="mailto:e.van.den.bout@amsterdam.nl"
-                >
+                <EmailLinkStyle href="mailto:e.van.den.bout@amsterdam.nl">
```

Comment:
It would be a lot better to not have any email addresses public and have them served from an Ansible vault. Maybe create a ticket for that?

Reply:
ticket added